bitnami/etcd/values.yaml

```diff
@@ -209,6 +209,12 @@ autoCompactionRetention: ""
 ## - 'existing': when upgrading the chart ('helm upgrade ...')
 ##
 initialClusterState: ""
+## @param initialClusterToken Initial cluster token. If spinning up multiple clusters (or creating and destroying a single cluster)
```

Comment:
The trailing text behind the `@param` (until the line break) is rendered in the `README.md` file automatically as the description of the parameter. 

Please, separate the definition of the parameter from further explanations in order to avoid incomplete/misleading descriptions:

https://github.com/bitnami/charts/blob/b7f34ce9aa44f1acd26b92171d858c9790d686b8/bitnami/etcd/README.md?plain=1#L117